pkg/proxy/proxy.go

```diff
@@ -57,7 +57,7 @@ type token struct {
 	RefreshToken string `json:"refresh_token"`
 
 	// AAD returns expires_in as a string, ADFS returns it as an int
-	ExpiresIn json.Number `json:"expires_in"`
+	ExpiresIn string `json:"expires_in"`
```

Comment:
@skythet As mentioned in the comment, this struct was vendor-ed from the go SDK: https://github.com/Azure/go-autorest/blob/b3899c1057425994796c92293e931f334af63b4e/autorest/adal/token.go#L1055-L1067. This has been the default behavior for a while.

What language SDK are you using? Also, another user opened https://github.com/Azure/azure-workload-identity/issues/773, which seems to indicate the proxy sidecar is working fine with their ODBC.

Reply:
@skythet ping! Did you get a chance to look at the comment?